happypose_examples/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.5.0)
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Is this the version of CMake that is the default on Ubuntu LTS 22.04?

Reply:
ref https://github.com/jrl-umi3218/jrl-cmakemodules/pull/620